test/torchaudio_unittest/models/tacotron2/model_test_impl.py

```diff
@@ -9,21 +9,28 @@
 class TorchscriptConsistencyMixin(TempDirMixin):
     r"""Mixin to provide easy access assert torchscript consistency"""
 
-    def _assert_torchscript_consistency(self, model, tensors):
+    def _assert_torchscript_consistency(self, model, tensors, is_inference=False):
```

Comment:
Instead of adding `is_inference`, can your wrap your model with `torch.nn.Module` and define `forward` that redirect to `infer` method?

Reply:
Thanks for the suggestion.
I've remove `is_inference` and use the wrapper implemented [here](https://github.com/pytorch/audio/pull/1648/files#diff-001793fb337f116c66a3e7f3869a18012a618ab079df93db381fec0f45b63d9eR9-R18).